### ISSUE TITLE
Change alt text for westside location image on events page

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#3255]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the westside location on [Events page](https://www.hackforla.org/events/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] For the file `_includes/events-page/our-locations-content.html`, change the alt text:

from
```
<img class="location-image" src="../assets/images/hack-nights/westside1.png" alt="Westside L.A."  />
```
to
```
<img class="location-image" src="../assets/images/hack-nights/westside1.png" alt="Westside" />
```
- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3132 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please move issue #3132 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3132 
- [Events page](https://www.hackforla.org/events/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/events-page/our-locations-content.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_includes/events-page/our-locations-content.html

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by JessicaLucindaCheng** [comment]
Should the alt text for this image be "Westside" or "Westside L.A." on the [Events page](https://www.hackforla.org/events/)?
<img width="323" alt="westside" src="https://user-images.githubusercontent.com/31293603/212781449-ffc644d3-d21a-4a74-85b1-b8c809392867.png">

Edit: Never mind. I found my answer. It should be "Westside" according to this [spreadsheet](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=253767243)

**Comment by KazushiR** [comment]
Availability for this week: Flexible
My estimated ETA for completing this issue: By end of this week.
